inc/genhdr/qstrdefs.generated.h

```diff
@@ -382,6 +382,19 @@ QDEF(MP_QSTR_PITCHFORK, (const byte*)"\x33\x09" "PITCHFORK")
 QDEF(MP_QSTR_XMAS, (const byte*)"\xe2\x04" "XMAS")
 QDEF(MP_QSTR_PACMAN, (const byte*)"\x55\x06" "PACMAN")
 QDEF(MP_QSTR_TARGET, (const byte*)"\xb4\x06" "TARGET")
+QDEF(MP_QSTR_TSHIRT, (const byte*)"\x45\x06" "TSHIRT")
+QDEF(MP_QSTR_ROLLERSKATE, (const byte*)"\x27\x0b" "ROLLERSKATE")
+QDEF(MP_QSTR_DUCK, (const byte*)"\x3c\x04" "DUCK")
+QDEF(MP_QSTR_HOUSE, (const byte*)"\xa1\x05" "HOUSE")
+QDEF(MP_QSTR_TORTOISE, (const byte*)"\xc8\x08" "TORTOISE")
+QDEF(MP_QSTR_BUTTERFLY, (const byte*)"\x56\x09" "BUTTERFLY")
+QDEF(MP_QSTR_STICKMAN, (const byte*)"\x01\x08" "STICKMAN")
+QDEF(MP_QSTR_GHOST, (const byte*)"\xc2\x05" "GHOST")
+QDEF(MP_QSTR_SWORD, (const byte*)"\x98\x05" "SWORD")
+QDEF(MP_QSTR_GIRAFFE, (const byte*)"\x7d\x07" "GIRAFFE")
+QDEF(MP_QSTR_SKULL, (const byte*)"\x48\x05" "SKULL")
+QDEF(MP_QSTR_BROLLY, (const byte*)"\x63\x06" "BROLLY")
```

Comment:
I did wonder. But I did take two attempts (speed typing) to type UMBRELLA so went for "brolly". :-) Actually Abbie calls it an umbrella so I'll stick with that. Change incoming.